Entity/Page.php

```diff
@@ -56,6 +56,11 @@ class Page
      */
     private $complementary_meta;
 
+    /**
+     * @ORM\Column(type="text", nullable=true)
+     */
+    private $rel_meta;
```

Comment:
There already a complementary_meta field inside of this bundle please use this one